src/collective/fingerpointing/browser/logview.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+from App.config import getConfiguration
+from collective.fingerpointing import _
+from collective.fingerpointing.config import AUDITLOG
+from Products.Five.browser import BrowserView
+from Products.Five.browser.pagetemplatefile import ViewPageTemplateFile
+
+import os
+import os.path
+
+
+# by default, the audit log will use the same location used for the event log
+eventlog = getattr(getConfiguration(), 'eventlog', None)
+
+# on tests, eventlog is not set; we need to handle that
+if eventlog is not None:
+    logpath = eventlog.handler_factories[0].instance.baseFilename
+    logfolder = os.path.split(logpath)[0]
+    logfile = os.path.join(logfolder, AUDITLOG)
+else:
+    logfile = AUDITLOG
+
+
+class LogView(BrowserView):
+
+    index = ViewPageTemplateFile('logview.pt')
+
+    def __call__(self):
+        self.show_all = False
+        if 'show_all' in self.request.form:
+            self.show_all = True
+
+        return self.render()
+
+    def render(self):
+        return self.index()
+
+    def label(self):
+        return _('Fingerpointing log view')
+
+    def tail(f, n=100):
```

Comment:
@pcdummy @frisi I replaced the tail function with this one because seems to me shorter and better.